flocker/node/agents/blockdevice.py

```diff
@@ -1198,13 +1266,14 @@ def create_volume(self, dataset_id, size):
         See ``IBlockDeviceAPI.create_volume`` for parameter and return type
         documentation.
         """
+        actual_size = allocated_size(self.allocation_unit(), size)
```

Comment:
Ok. I removed the rounding in LoopbackBlockDeviceAPI and raise a ``ValueError`` if the supplied size isn't divisble.
That broke a bunch of other tests, which I've now fixed by replacing REALISTIC_BLOCKDEVICE_SIZE with ``allocation_unit()``.
Also lowered the default ``LoopbackBlockDeviceAPI.allocation_unit()`` to 64MiB in line with our minimum API size which is based on ZFS contraints....I think.